### DISCUSSION TITLE
How to use LazyVim.pick() in my config?

### DISCUSSION THREAD

**Discussion by ada4a** [LazyVim/LazyVim#3581] [Q&A] [answered]
I saw that in a [couple of places](https://github.com/search?q=repo%3ALazyVim%2FLazyVim+pick%28&type=code) in the default config, a handy function called `LazyVim.pick()` is used, which makes for a shorter version of `LazyVim.telescope()`. I wanted to incorporate it into my config like this:
```
-  { "<leader>fd", LazyVim.telescope("find_files", { cwd = "~/.dotfiles" }), desc = "Find Dotfiles" },
+  { "<leader>fd", LazyVim.pick({ cwd = "~/.dotfiles" }), desc = "Find Dotfiles" },
```
... but unfortunately, when I restart Neovim after applying this change, I get the following error:
```
Failed to load `plugins.telescope`

....local/share/nvim/lazy/LazyVim/lua/lazyvim/util/init.lua:51: module 'lazyvim.util.pick' not found:
	no field package.preload['lazyvim.util.pick']
cache_loader: module lazyvim.util.pick not found
cache_loader_lib: module lazyvim.util.pick not found
	no file '/nix/store/ivb32bi4wk08xi6zjvh7wp3iyj7qq0l3-luajit-2.1.1693350652-env/share/lua/5.1/lazyvim/util/pick.lua'
	no file '/nix/store/ivb32bi4wk08xi6zjvh7wp3iyj7qq0l3-luajit-2.1.1693350652-env/share/lua/5.1/lazyvim/util/pick/init.lua'
	no file '/nix/store/ivb32bi4wk08xi6zjvh7wp3iyj7qq0l3-luajit-2.1.1693350652-env/lib/lua/5.1/lazyvim/util/pick.so'
	no file '/nix/store/ivb32bi4wk08xi6zjvh7wp3iyj7qq0l3-luajit-2.1.1693350652-env/lib/lua/5.1/lazyvim.so'

# stacktrace:
  - /LazyVim/lua/lazyvim/util/init.lua:51 _in_ **__index**
  - .config/nvim/lua/plugins/telescope.lua:4
  - .config/nvim/lua/config/lazy.lua:10
  - .config/nvim/init.lua:2
```
... which is weird, because the function does seem to be defined [here](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/util/pick.lua) and is, as I said, currently used, so it shouldn't be deprecated or something. What am I missing here?

Thanks in advance:)

**Comment by dpetka2001** [comment, marked as answer]
You're probably on LazyVim stable release. This module is only available on latest HEAD for now. Either switch to latest HEAD or wait for the next LazyVim release.
